app/views/guides/new/_new_guide_step_1.html.erb

```diff
@@ -67,7 +67,7 @@
           <div class="columns small-12">
             <span class="thumbnails"
                   ng-if="newGuide.featured_image && newGuide.featured_image.indexOf('leaf-grey') === -1">
```

Comment:
Correction: unicode character is actually just two highlighted ampersands :grimacing: